response_operations_ui/views/messages.py

```diff
@@ -109,7 +110,7 @@ def view_conversation(thread_id):
             thread_url = url_for("messages_bp.view_conversation", thread_id=thread_id) + "#latest-message"
             flash(Markup(f'Message sent. <a href={thread_url}>View Message</a>'))
             return redirect(url_for('messages_bp.view_selected_survey',
-                                    selected_survey=refined_thread[0]['survey']))
+                                    selected_survey=refined_thread[0]['survey'].replace(' ', '')))
```

Comment:
why doesn't this use the replace whitespace function?

Reply:
Done